### ISSUE TITLE
Pinned Post Always on Top?

### ISSUE THREAD

**Issue by donatj** [ciderpunx/twitrssme#51]
It would appear the authors pinned post is always on top of the feed.

Could we have an option to not include this post outside the normal feed of posts? I'd rather have a feed of posts as they happened.

**Comment by bmomjian** [comment]
Agreed, I would like this too.  I don't see anything in the RSS that would indicate the post is pinned --- that would be another option.  Adding to the 'p class'  with a pin indication would make sense.

**Comment by donatj** [comment]
I'd think you could check the pubDate of the first and second post, and if they're out of order the first one is pinned?

**Comment by bmomjian** [comment]
Great idea, and it worked.  Thanks.

**Comment by micressor** [comment]
@bmomjian Would it be possible for you to provide a diff until it is in the master branch?

**Comment by bmomjian** [comment]
I don't have  a patch --- I ended up fixing it in my application.  I started at the first tweet and advanced as long as the next tweet's pubdate  was _later_ than the current one.  I assumed all of those were pinned.  This would not work if multiple entries were pinned, and a newer entry was pinned before a later entry, but I don't have that case with the feed I am monitoring.
